### PR TITLE
Add additional JetStream advisories

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

File: surveyor/jetstream.go [hunk 1]
```diff
@@ -82,6 +82,11 @@ var (
 		Help: "Advisories about JetStream Consumer Delivery Exceeded events",
 	}, []string{"account", "stream", "consumer"})
 
+	jsDeliveryTerminatedCtr = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: prometheus.BuildFQName("nats", "survey", "jetstream_delivery_terminated_count"),
+		Help: "Advisories about JetStream Consumer Delivery Terminated events",
+	}, []string{"account", "stream", "consumer"})
```
Comment (ColinSullivan1) [inline surveyor/jetstream.go:88]: I just noticed this...   Should the JS labels mirror the `Statz` labels?

{"server_cluster", "server_name", "server_id", "account", "stream", "consumer"}

**Comment by ripienaar** [comment]
I am not sure how a clustered JS will behave and if the server ID really matters then 

Besides that I think server id indexing is a mistake since it’s dynamic and non deterministic. Name would be better if name had a sane default other than ID.

**Comment by ripienaar** [comment]
I am not sure how a clustered JS will behave and if the server ID really matters then 

Besides that I think server id indexing is a mistake since it’s dynamic and non deterministic. Name would be better if name had a sane default other than ID.

File: docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ services:
     volumes:
       - $NATS_SURVEYOR_CREDS:/etc/surveyor/SYS.creds
       - ./observations:/observations
+      - ./jetstream:/jetstream
```
Comment (ColinSullivan1) [inline docker-compose/docker-compose.yml:15]: I'm wondering if the storage here should be consolidated.  One volume with obervations.json, jetstream.json, etc, or even consolidate into one config file.  wdyt?

**Comment by ripienaar** [comment]
Separate files are easier to maintain with automation tools and files are simpler

**Comment by ColinSullivan1** [comment]
Makes sense.  I'd bet server_name will be useful (e.g. correlating server cpu/memory to JS stats) but we can always add to later as necessary.  ...would rather add later than have superfluous stats.

**Pull request merged**